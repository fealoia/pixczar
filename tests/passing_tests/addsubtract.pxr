Void main() {
	Int x = 4;
	Int y = 3;
	print(x+y);
	print(x-y);

	Float a = 3.2;
	Float b = 3.1;
	print(a+b);
	print(a-b);
}
