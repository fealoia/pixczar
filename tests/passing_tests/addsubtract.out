7
1
6.3
0.1
